menu-mobile.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+  <title>mobile-menu</title>
+  <link href="style-mobile-menu.css" rel="stylesheet">  
+</head>
+
+<body>
+  
+  
+  <div class="mobile-menu">
+       <div class="cross">
+          <a href="index.html"><img src="./images/cross.png"></a>
+        </div>   
+       <div class="entries"> <a href="./index.html#portfolio">Portfolio</a> </div>
+       <div class="entries"> <a href="./index.html#about"> About</a> </div>
+       <div class="entries" class="contact">  <a href="./index.html#contact"> Contact</a> </div>
+  </div>
+  
+</body>
+
+
+</html>
```

Comment:
- [ ] Please know that you shouldn't have a new HTML file just for the mobile menu, this caused some issues and I wasn't able to see the menu at all here is what I see on the mobile screen:
  ![image](https://user-images.githubusercontent.com/20567503/196307903-8570800e-6628-41c8-9427-133d1274bba8.png)
  Kindly fix this issue so we can test your project.